client/src/components/Card.jsx

```diff
@@ -1,18 +1,11 @@
 import React, { useState, useEffect } from 'react';
-import axios from 'axios';
 
-export default function Card () {
-  const [productInfo, setProductInfo] = useState([])
 
-  const getProductInfo = () => {
-    axios.get('/product/productid')
-      .then((rawData) => {
-        setProductInfo([rawData.data])
-      })
-      .catch((err) => {
-        console.log('Error getting Product...', err)
-      })
-  }
+export default function Card (props) {
```

Comment:
Just learned that you can export default named functions if you use the traditional function declaration with the 'function' keyword. Interesting!